views/help/partials/help_confluence_links.hbs

```diff
@@ -1,15 +1,15 @@
 {{#embed "help/icon-card" title="Inhaltsverzeichnis" icon="fa-list" class="table-of-contents-card hidden-sm-up col-xs-12" id="inhaltsverzeichnis"}}
 	{{#content "card-body"}}
-	<ul class="white-links">
```

Comment:
merging error? still present on develop...

Reply:
yep probably, this come not from my changes. Fixed ;)